### PR TITLE
Minify HTML even more!

### LINKED ISSUE #11: Optionally remove whitespace after vulcanization of css
> 
As the vulcanize process inlines the JS and CSS for elements, the resulting elements.vulcanized.html contains several hundred KB of empty space. By using gulp-minify-inline on the vulcanized file, the size goes from 447K down to around 267K for the default install. Even with gzipped calls, this can be quite a bit of pageload reduced.

ref:
https://github.com/PolymerElements/polymer-starter-kit/pull/486#issue-115633232
https://github.com/PolymerElements/polymer-starter-kit/issues/564
https://github.com/PolymerElements/polymer-starter-kit/pull/565
https://github.com/Polymer/vulcanize/issues/283

### PR DESCRIPTION
**Pull request by azakus**
Fixes #11 Optionally remove whitespace after vulcanization of css

Forward collapseWhitespace option with `--html.collapseWhitespace`

### CODE REVIEW

File: src/build/optimize.ts [hunk 1]
```diff
@@ -25,8 +25,7 @@ export interface OptimizeOptions {
    * Options passed to gulp-html-minifier
    */
   html?: {
-    collapseWhitespace?: boolean;
-    removeComments?: boolean;
+    [property: string]: boolean;
```
Comment (justinfagnani) [inline src/build/optimize.ts]: revert?

File: src/commands/build.ts [hunk 2]
```diff
@@ -40,18 +41,29 @@ export class BuildCommand implements Command {
...
     // Defer dependency loading until this specific command is run
     const build = require('../build/build').build;
 
-    let buildOptions = {
+    let buildOptions: BuildOptions = {
       sources: options.sources,
       swPrecacheConfig: options['sw-precache-config'],
-      insertDependencyLinks: options['insert-dependency-links']
+      insertDependencyLinks: options['insert-dependency-links'],
+      html: {},
```
Comment (justinfagnani) [inline src/commands/build.ts]: are these required? If so, update the OptimizeOptions interface.

**Comment by justinfagnani** [comment]
LGTM

File: src/commands/build.ts (hunk 2 above)
Comment (azakus) [inline src/commands/build.ts]: They are not required, I just didn't want to make them conditionally based on the flags.

**Pull request merged**